run/john.conf

```diff
@@ -745,6 +745,9 @@ l Az"[1-90][0-9][0-9]" <+
 -c (?a c Az"[1-90][0-9][0-9]" <+
 <[\-9] l A\p[z0]"[a-z][a-z]"
 <- l ^[a-z] $[a-z]
```

Comment:
Thanks! You are currently adding the new rules to `[List.Rules:Extra]`. For me to merge it, I'd want you to instead create a new section eg. `[List.Rules:T9]`. Please do that and I'll merge it.

Reply:
Ok done. Re: Fast hash vs slow hash. I've found a fair amount (few dozen) of 14-16+ digits long passwords (from Troy Hunt's pwned pass file) with that rule and big dictionaries and I hadn't pushed my Digits brute force above 10 chars long 😅